src/main/java/pl/touk/service/DriverService.java

```diff
@@ -0,0 +1,66 @@
+package pl.touk.service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import org.springframework.util.StringUtils;
+import pl.touk.model.Driver;
+import pl.touk.repository.DriverRepository;
+import pl.touk.service.dto.NewDriverConverter;
+import pl.touk.service.dto.NewDriverDto;
+import pl.touk.service.dto.UpdateDriverConverter;
+import pl.touk.service.dto.UpdateDriverDto;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class DriverService {
+
+    private final DriverRepository driverRepository;
```

Comment:
ile zależności w klasie to byłoby dla Ciebie za dużo?

Reply:
Myślę, że 10 zaleźności to byłoby za dużo. Wydaje mi się, ze optymalnie do 5. Jednak ważną kwestią jest czytelność konstruktora. Konstruktor, który ma np. 10 czy 20 zmiennych nie byłby łatwy do analizy, bo np. nie mieści się na ekranie.